container/Dockerfile

```diff
@@ -0,0 +1,65 @@
+# This dockerfile creates a container of a truly basic DIRAC installation which contains only the DIRAC externals.
+# The sole purpose of this container is for developing (and testing) DIRAC "stuff that run",
+# meaning DIRAC components like agents and services. Instructions in ...
+
+# THIS IS A WORK IN PROGRESS
+# TODO:
+#   verify about certificates
+#   correct openssl_config URL location
+#   lcgBundle now fix version
+#   start mySQL + install/start ElasticSearch (?) + install/start ActiveMQ (?) or better use docker-compose
+#
+
+
+# CC7 as base (the latest tag doesn't work!)
+FROM cern/cc7-base:20170113
+MAINTAINER Federico Stagni <federico.stagni@cern.ch>
+
+# Create DIRAC dirs
+RUN mkdir -p /opt/dirac/DIRAC && \
+    mkdir -p /opt/dirac/etc/grid-security/certificates && \
+    mkdir -p /opt/dirac/user && \
+    cd /opt/dirac
+
+# Installing DIRAC in /opt/dirac
+RUN curl -L -o dirac-install https://raw.githubusercontent.com/DIRACGrid/DIRAC/integration/Core/scripts/dirac-install.py && \
+    chmod +x dirac-install && \
+    ./dirac-install -r v6r17 -t server -i 27 -g 2017-01-27 && \
+    rm -rf /opt/dirac/.installCache && \
+    rm dirac-install
+
+# Copying in ENTRYPOINT script (for running DIRAC scripts directly, e.g. dirac-service)
+COPY dockerEntrypoint.sh /opt/dirac/dockerEntrypoint.sh
+RUN chmod 755 /opt/dirac/dockerEntrypoint.sh
+ENTRYPOINT [ "/opt/dirac/dockerEntrypoint.sh" ]
+
+# Create self-signed host certificate from auto-generated CA
+WORKDIR /opt/dirac/etc/grid-security
+RUN source /opt/dirac/bashrc && openssl genrsa -out hostkey.pem 2048 2&>1 /dev/null
```

Comment:
what is the /dev/null for ? You want to silence all the output ? ``` &> /dev/null ``` is your friend